libraries/provision/ansible/playbooks/files/sysctld_disable_memory_overcommit.conf

```diff
@@ -0,0 +1,9 @@
+
+# Disable memory overcommit, which will eliminate cases where the Linux OOM (Out-Of-Memory) killer kills
+# processes, which was seen in https://github.com/couchbaselabs/sync-gateway-accel/issues/78.
+# Do this by default since it is easier to debug things when a process crashes because it can't malloc (allocate)
+# more memory, rather than the Linux OOM killer asynchronously killing it
+sysctl vm.overcommit_memory=2
+
+# This tells Linux to keep 20% of the OS memory to itself
+sysctl vm.overcommit_ratio=80
```

Comment:
Yes because it's in a `.conf` file that's being copied over to the `/etc/sysctl` directory